frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -121,11 +121,16 @@ export function SurveyReleaseSummary({
     survey: Survey | NewSurvey
     targetingFlagFilters?: FeatureFlagFilters
 }): JSX.Element {
+    const hasConditions = survey.conditions?.url || survey.conditions?.selector
```

Comment:
For a bonus, would be cool if we can add a line for the wait period here as well, since technically if wait period is set it may not show to everyone.

Reply:
ooh, great context!  Will add.

Reply:
It also requires adding a bit of text, right now I think when wait period is set the summary shows up empty 😅

Reply:
oh man, I'm so sorry I missed that.  Out of character for me; I should've double-checked.  Here's a fix: https://github.com/PostHog/posthog/pull/22532